osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -225,6 +244,9 @@ private double computeSpeedValue(ScoreInfo score, OsuDifficultyAttributes attrib
                 speedValue *= 1.0 + 0.04 * (12.0 - attributes.ApproachRate);
             }
 
+            double speedHighDeviationMultiplier = calculateSpeedHighDeviationNerf(attributes);
+            speedValue *= speedHighDeviationMultiplier;
+
             // Calculate accuracy assuming the worst case scenario
```

Comment:
Is this even necessary now? Seems like its doing the same thing, except worse

Reply:
what do you mean by "is this necessary"
it's a line that's applying the nerf to speed pp

Reply:
My comment is on the line 250, im talking about worst case accuracy scaling below

Reply:
Oh, ok
It's necessary because it's normal speed scaling from accuracy
Additional scaling added in this PR is adding new scaling for cases where tapping is not normal (aka raking or doubletapping), that's nerfing score in additional to normal scaling from accuracy